src/wasm/wasm-stack.cpp

```diff
@@ -1953,19 +1954,40 @@ void BinaryInstWriter::visitTry(Try* curr) {
   emitResultType(curr->type);
 }
 
+void BinaryInstWriter::visitTryTable(TryTable* curr) {
+  // the binary format requires this; we have a block if we need one
```

Comment:
This comment relates to line 1975, I think? Perhaps move it down if so?

Reply:
Right, I copy-pasted it from `visitIf` and left it in a wrong place....